### PR TITLE
RHBPMS-1889: Search by Find form does not work.

### PR DESCRIPTION
**Pull request by manstis**
Fix failing tests on kie-wb-common and include test in uberfire.

### CODE REVIEW

**Comment by manstis** [comment]
This is related to https://issues.jboss.org/browse/RHBPMS-1889 and https://github.com/uberfire/uberfire/pull/618

**Comment by manstis** [comment]
@psiroky This fixes the failing tests on ```kie-wb-common```

File: uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/IOServiceIndexedDeleteFileTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                                  return "delete";
+                              }
+
+                              @Override
+                              public Object value() {
+                                  return "me";
+                              }
+                          });
+
+        Thread.sleep(5000); //wait for events to be consumed from jgit -> (notify changes -> watcher -> index) -> lucene index
```
Comment (ederign) [inline uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/IOServiceIndexedDeleteFileTest.java]: In future, we should refactoring lucene index api to trigger a signal when the index is ready!

File: uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/IOServiceIndexedImpl.java [hunk 2]
```diff
@@ -269,10 +269,16 @@ public void run() {
                             // Get a set of "real paths" to be indexed. The "dot path" associated with the "real path"
                             // is automatically indexed because the "dot path" contains content for FileAttributeView(s)
                             // linked to the "real path".
-                            final Set<Path> eventRealPaths = events.stream().map((event) -> {
+                            final Set<Path> eventRealPaths = new HashSet<>();
+                            for (WatchEvent event : events) {
                                 final WatchContext context = ((WatchContext) event.context());
-                                return context.getPath();
-                            }).filter((path) -> !path.getFileName().toString().startsWith(".")).collect(Collectors.toSet());
+                                if (event.kind() == ENTRY_MODIFY || event.kind() == ENTRY_CREATE) {
```
Comment (ederign) [inline uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/IOServiceIndexedImpl.java]: just for the sake of my curiosity, is there any reason to move from streams to a for?

**Comment by ederign** [comment]
Thanks @manstis !

File: uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/IOServiceIndexedImpl.java (hunk 2 above)
Comment (manstis) [inline uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/IOServiceIndexedImpl.java]: @ederign IMO the filter would have become messy and using the loop made the code clearer.

**Pull request merged**